_includes/code/howto/search.filters.ts

```diff
@@ -261,16 +261,22 @@ for (let object of result.objects) {
 // }
 }
 
+let targetId = await jeopardy.data.insert({
+  properties: {
+    question: 'This is a temporary object to demonstrate deletion',
+    answer: 'EphemeralObject',
+    round: 'Jeopardy!',
+    points: 100,
+  }
+})  
 // ===================================================
 // ===== Filters using Id =====
 // ===================================================
 {
 // filterById
-const targetId = '00037775-1432-35e5-bc59-443baaef7d80'
-
-result = await article.query.fetchObjects({
+let result = await jeopardy.query.fetchObjects({
```

Comment:
we don't necessarily have to redefine the variable with let because we already defined it above; we can just assign it a value now throughout the rest of the file.